.circleci/config.yml

```diff
@@ -96,6 +113,18 @@ jobs:
               echo "$DOCKER_HUB_PASSWORD" | docker login -u "$DOCKER_USER" --password-stdin
               ./mvnw spring-boot:build-image -Dspring-boot.build-image.imageName=ehrbase/aql-editor:$TAG -Dmaven.test.skip
               docker push ehrbase/aql-editor:$TAG
+    
+    merge_to_nightly_build:
+      docker:
+            - image: circleci/openjdk:stretch
+      steps:
+        - run:
+              name: Merge master to nightly-builds branch
+              command: |
+                      - git checkout master
```

Comment:
I don't think git checkout master and git pull should be needed, the code should be the latest when this branch is ran. Furthermore, you are not pushing your changes at all here.